File: Python/Nested if.py
# Nested if ....

is_egyptian = input("Are You Egyptian ? Type (Yes) or (No) ").lower() 
if is_egyptian == 'yes' :
    print("Good , You Can GO To The Next Step")

    is_18 = ((input("Are You above 18 ? Type (Yes) or (No) ")).lower() )

    if is_18 == 'yes' :
        print("You Can Have ID")
    else :
        print("Sorry , you have to be 18 or older \nplease try again when you are 18")

else :
    print("Sorry , an Egyptian ID Is GIven To Egyptian.., Thank You")
    

File: C++/Basic/Nested If.cpp
//  Neasted if

#include <iostream>
using namespace std;
int main(){
int a =100 ;
int b =200 ;

if (a==100)
{
    if (b=200)      //! Neasted if 
    {
        cout<<"value of a is 100 and b is 200"<<endl;
    }
    else if (b==100)
    {
        cout<< "hi salah"<<endl;
    }
    else 
    {
        cout<< "thank you"<<endl;
    }

}

return 0;
}